templates/coredata/swift4.stencil

```diff
@@ -179,7 +179,7 @@ extension {{ entityClassName }} {
     let model = persistentStoreCoordinator.managedObjectModel
     let substitutionVariables: [String: Any] = [
       {% for variableName, variableType in fetchRequest.substitutionVariables %}
-      "{{ variableName }}": {{ variableName | lowerFirstWord }}
+      "{{ variableName }}": {{ variableName | lowerFirstWord }}{{ ", " if not forloop.last }}
```

Comment:
TIL that Stencil supported the `{{ x if y }}` ruby-like construct, way more compact than the `{% if not forloop.last %}, {% endif %}`, I missed that, 👍 

We should update all other templates to use that construct instead – for those not already using it that is, i.e. plist, json, yaml, and the strings-objc ones

Reply:
Fixed in https://github.com/SwiftGen/SwiftGen/pull/693